udata/templates/macros/metadata.html

```diff
@@ -46,8 +46,8 @@
 />
 {% endif %}
 
-<meta name="check-urls" content="{{ 'true' if config.CROQUEMORT else 'false' }}" />
-<meta name="check-urls-ignore" content="{{ config.CROQUEMORT_IGNORE|tojson|urlencode }}" />
+<meta name="check-urls" content="{{ 'true' if config.LINKCHECKING_ENABLED else 'false' }}" />
```

Comment:
Or `<meta name="check-urls" content="{{ config.LINKCHECKING_ENABLED|to_json }}" />`
Works for other booleans.